migrate-unpack.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/bash
+echo "Restore /app/imports/nidirectd7.sql.gz into drupal7db database..."
+gunzip /app/imports/nidirectd7.sql.gz
```

Comment:
I don't like this very much, but the more efficient (unpacks into a stream to the mysql client) `gzip -dc /app/imports/nidirectd7.sql.gz | drush -r /app/web sqlc --database=drupal7db` seemed to go dead at some point along the way. Possibly the stream was interrupted by something, it's not clear.

Either way, a manual unpack + restore + tidyup works fine.